src/components/FoodForm.js

```diff
@@ -43,27 +42,34 @@ const FoodForm = () => {
 			calories: 375,
 		},
 	]);
+
+	function foodSearch(myQuery) {
+		fetch(`http://localhost:3000/api/food/recipes?query=${myQuery}&number=1&max_fat=40`)
+			.then(response => response.json())
+			.then(json => console.log(json));
```

Comment:
you are logging the results here, but nothing more. 
this is not a finished functionality.

Reply:
Youre right im working on it. not finished yet.